README.md

```diff
@@ -1,290 +1,4 @@
-
-### Get info on a video (width, height, orientation, duration, size, & bitrate)
-```javascript
-VideoEditor.getVideoInfo(
-    success, // success cb
-    error, // error cb
-    {
-        fileUri: 'file-uri-here', // the path to the video on the device
-    }
-);
-```
-
-```javascript
-VideoEditor.getVideoInfo(
-    getVideoInfoSuccess,
-    getVideoInfoError,
-    {
-        fileUri: file.fullPath
-    }
-);
-
-function getVideoInfoSuccess(info) {
-    console.log('getVideoInfoSuccess, info: ' + JSON.stringify(info, null, 2));
-    // info is a JSON object with the following properties -
-    {
-        width: 1920,
-        height: 1080,
-        orientation: 'landscape', // will be portrait or landscape
-        duration: 3.541, // duration in seconds
-        size: 6830126, // size of the video in bytes
-        bitrate: 15429777 // bitrate of the video in bits per second
-    }
-}
-```
-
-## Android & FFmpeg
-FFmpeg has been removed from android for several reasons but mainly for performance.  If you still need the old functionality that FFmpeg provided  [V1.09](https://github.com/jbavari/cordova-plugin-video-editor/tree/1.0.9) is the last version that will use it.
-
-## On iOS
-
-[iOS Developer AVFoundation Documentation](https://developer.apple.com/library/ios/documentation/AudioVideo/Conceptual/AVFoundationPG/Articles/01_UsingAssets.html#//apple_ref/doc/uid/TP40010188-CH7-SW8)
-
-[Video compression in AVFoundation](http://www.iphonedevsdk.com/forum/iphone-sdk-development/110246-video-compression-avassetwriter-in-avfoundation.html)
-
-[AVFoundation slides - tips/tricks](https://speakerdeck.com/bobmccune/composing-and-editing-media-with-av-foundation)
-
-[AVFoundation slides #2](http://www.slideshare.net/bobmccune/learning-avfoundation)
-
-[Bob McCune's AVFoundation Editor - ios app example](https://github.com/tapharmonic/AVFoundationEditor)
-
-[Saving videos after recording videos](http://stackoverflow.com/questions/20902234/save-video-to-library-after-capturing-video-using-phonegap-capturevideo)
-
-
-
-## On Android
-
-[Android Documentation](http://developer.android.com/guide/appendix/media-formats.html#recommendations)
-
-[Android Media Stores](http://developer.android.com/reference/android/provider/MediaStore.html#EXTRA_VIDEO_QUALITY)
-
-[How to Port ffmpeg (the Program) to Android–Ideas and Thoughts](http://www.roman10.net/how-to-port-ffmpeg-the-program-to-androidideas-and-thoughts/)
-
-[How to Build Android Applications Based on FFmpeg by An Example](http://www.roman10.net/how-to-build-android-applications-based-on-ffmpeg-by-an-example/)
-
-
-## On Windows
-
-
-## License
-
-Android: Apache 2.0
-
-iOS: MIT
-
-Windows: Apache 2.0
+基于cordova-plugin-video-editor@1.1.3版本 
```

Comment:
You should probably not change the README.md file.

Reply:
Sorry, My mistake. Will you merge the code? I will close this request, revise and pull a new request. Thank you。